### PR TITLE
Enable dependabot additional updates.

### PR DESCRIPTION
**Pull request by dekiel**
**Description**

Changes proposed in this pull request:

- Removed ginkgo and gomega from ignored list.
- Added location with dockerfiles for log scanner app.

### CODE REVIEW

**Review by Sawthis — APPROVED** [review]

File: templates/data/dependabot-config-data.yaml [hunk 1]
```diff
@@ -169,3 +156,45 @@ templates:
...
+                  directory: "/development/gcp/cloud-run/move-gcs-bucket"
+                  # commit message can be 15 characters max
+                  commitMsgPrefix: "move-gcs-bucket"
+                inheritedConfigs:
+                  local:
+                    - "dailyDockerEcosystem"
+              - jobConfig:
+                  directory: "/development/gcp/cloud-run/slack-message-sender"
+                  # commit message can be 15 characters max
+                  commitMsgPrefix: "move-gcs-bucket"
```
Comment (Sawthis) [inline templates/data/dependabot-config-data.yaml:190]:
```suggestion
                  commitMsgPrefix: "slack-msg-sender"
```

File: templates/data/dependabot-config-data.yaml [hunk 2]
```diff
@@ -169,3 +156,45 @@ templates:
...
+                inheritedConfigs:
+                  local:
+                    - "dailyDockerEcosystem"
+              - jobConfig:
+                  directory: "/development/gcp/cloud-run/slack-message-sender"
```
Comment (Sawthis) [inline templates/data/dependabot-config-data.yaml:195]: slack-message-sender is defined twice on the ignore list

**Review by Sawthis — CHANGES_REQUESTED** [review]

File: templates/data/dependabot-config-data.yaml (hunk 1 above)
Comment (dekiel) [reply templates/data/dependabot-config-data.yaml]: Corrected.

File: templates/data/dependabot-config-data.yaml (hunk 2 above)
Comment (dekiel) [reply templates/data/dependabot-config-data.yaml]: Removed duplicated entries.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -54,17 +48,6 @@ updates:
     commit-message:
       prefix: "githubrelease"
       include: "scope"
-  - package-ecosystem: "docker"
-    directory: "/development/kyma-ci-force-bot/ciforcebot"
```
Comment (Sawthis) [inline .github/dependabot.yml:58]: Why have you removed ciforcebot?

Comment (dekiel) [reply .github/dependabot.yml:58]: Ciforcebot was never used. It's a leftover from POC.

**Review by Sawthis — APPROVED** [review]

**Pull request merged**